### ISSUE TITLE
> ### Github tags: Gbemiga01

### ISSUE THREAD

**Issue by Gbemiga01** [microsoft/opensource.microsoft.com#231]
> ### Github tags:
> #A11yMAS; #A11ySev2; #A11yTCS; #Benchmark; #P2_OSPOProduction_Web_Sep2021; #P2_WCP; #WCP; #OSPO Production; #DesktopWeb; #ChromiumEgde; #Win11; #P2_OSPOProduction; #WCAG1.4.3; #color-contrast;
> 
> ### Environment and OS Details:
> #URL: https://opensource.microsoft.com/codeofconduct/faq/ App name: OSPO Production Windows 11 Version 21H2(OS Build 22000.795) Chromium Edge Version 103.0.1264.62 (Official build) (64-bit)
> 
> ### Note:
> This issue is also observed in https://opensource.microsoft.com/program/
> 
> ### Issue:
> Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (color-contrast - https://accessibilityinsights.io/info-examples/web/color-contrast)
> 
> ### Target application:
> Code of Conduct - FAQ | Microsoft Open Source - https://opensource.microsoft.com/codeofconduct/faq/
> 
> ### Element path:
> li:nth-child(2) > span
> 
> ### Snippet:
> span
> 
> ### How to fix:
> Fix any of the following: Element has insufficient color contrast of 1.46 (foreground color: #323130, background color: #11100f, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1
> 
> Environment: Microsoft Edge version 103.0.1264.71
> 
> ====
> 
> This accessibility issue was found using Accessibility Insights for Web 2.32.0 (axe-core 4.4.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
> 
> ### Attachments:
> ![Code of Conduct - FAQ_AI4W](https://user-images.githubusercontent.com/83570638/181730125-1b2fb5e1-178e-4014-8167-7d10b1351691.png) ![Microsoft’s Open Source Program_AI4W_color-contrast](https://user-images.githubusercontent.com/83570638/181730168-024515f2-d543-4d60-94ae-72fd65437ead.png)

_Originally posted by @Gbemiga01 in https://github.com/microsoft/opensource.microsoft.com/issues/219#issuecomment-1257226368_

**Comment by SachinChavhan317** [comment]
I have verified the bug on URL: https://opensource.microsoft.com/program/ , Observed that issue is not reproducing. Hence closing the bug. #Closed
Added below screenshot for reference.

![231_Bug_Fixed_Note](https://user-images.githubusercontent.com/86241494/216950719-85ef61e9-87a6-410b-8ea8-5dd5d7b513ce.png)


![231_Bug_Fixed](https://user-images.githubusercontent.com/86241494/216950191-c8da5888-1ddb-453a-9383-ad48f52917c8.png)

**Comment by Samhitha66** [comment]
#Closed;
